modules/kmm-configuring-kmmo.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="kmm-configuring-kmmo_{context}"]
+= Configuring the Kernel Module Management Operator
+
+You can configure the Kernel Module Management (KMM) Operator out of the box with sensible defaults.
+The Operator configuration is set in the `kmm-operator-manager-config` `ConfigMap` in the Operator namespace.
+
+.PROCEDURE
+
+. To modify the settings, edit the `ConfigMap` data, as shown in the following example.
++
+[source,yaml]
+----
+healthProbeBindAddress: :8081
+leaderElection:
+  enabled: true
+  resourceID: kmm.sigs.x-k8s.io
+webhook:
+  disableHTTP2: true  # CVE-2023-44487
+  port: 9443
+metrics:
+  enableAuthnAuthz: true
+  disableHTTP2: true  # CVE-2023-44487
+  bindAddress: 0.0.0.0:8443
+  secureServing: true
+worker:
+  runAsUser: 0
+  seLinuxType: spc_t
+  setFirmwareClassPath: /var/lib/firmware
+----
++
+.KMM configuration parameters
```

Comment:
@qbarrand Is this correct? Should it be "KMM Operator configuration parameters"?

Reply:
Let us use "Operator configuration parameters" since we are already in the KMM docs.